cmd/datamon/cmd/flags.go

```diff
@@ -454,10 +454,6 @@ func paramsToContributor(flags flagsT) (model.Contributor, error) {
 	default:
 		credentials = os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")
 	}
-	if credentials == "" {
-		return model.Contributor{},
-			fmt.Errorf("could not resolve credentials: must be present as --credential flag, or in local config or as GOOGLE_APPLICATION_CREDENTIALS environment")
-	}
```

Comment:
@ritesh yes I guess you are right. I am curious about what the other ways are.

Reply:
SDK also looks in the ~/.config folder. So the cred file setup is not required if you have setup application default creds file in ~/.config. I will drop the environment check from the function as well.